migration-engine/migration-engine-tests/tests/migration_tests.rs

```diff
@@ -1968,10 +1968,10 @@ async fn join_tables_between_models_with_compound_primary_keys_must_work(api: &T
             .assert_has_column("cat_id")?
             .assert_fk_on_columns(&["human_firstName", "human_lastName"], |fk| {
                 fk.assert_references("Human", &["firstName", "lastName"])?
-                    .assert_cascades_on_delete()
+                    .assert_restrict_on_delete()
```

Comment:
I am not sure why this doesn't fail for SQL Server as we are using `NO ACTION` for this database.

Reply:
It did fail. I just did not push the latest version :-D